mdagent/tools/base_tools/simulation_tools/create_simulation.py

```diff
@@ -0,0 +1,180 @@
+import textwrap
+from typing import Optional
+
+import langchain
+from langchain import LLMChain, PromptTemplate
+from langchain.base_language import BaseLanguageModel
+from langchain.tools import BaseTool
+from pydantic import BaseModel, Field
+
+from mdagent.utils import PathRegistry
+
+
+class ModifyScriptUtils:
+    llm = langchain.chat_models.ChatOpenAI(
```

Comment:
@Jgmedina95 can we make the llm either inherit the model type from tools llm or be an input?

Reply:
done!